src/components/Description/index.tsx

```diff
@@ -0,0 +1,104 @@
+import { css } from '@emotion/react'
+import styled from '@emotion/styled'
+import PropTypes from 'prop-types'
+import React, { FunctionComponent } from 'react'
+import Box, { BoxProps } from '../Box'
+
+const StyledDt = styled(Box)``
+const StyledDd = styled(Box)``
```

Comment:
🤯 Why not something like:
```suggestion
const StyledDt = styled(Box.withComponent('dt'))`
    font-weight: 500;
    color: ${({ theme }) => theme.colors.gray950};
    &:after {
      content: ':';
    }
  `
const StyledDd = styled(Box.withComponent('dd'))`
    color: ${({ theme }) => theme.colors.gray700};
    margin: 0;
    margin-top: ${({ theme }) => theme.space['1']};
`
```